oasislmf/utils/inputs.py

```diff
@@ -110,31 +119,53 @@ def get(self, name, default=None, required=False, is_path=False):
             use config_dir as base dir if value comes from the config
         :type is_path: bool
 
+        :param dtype: the class <type> of the value, if 'None' load as string by default
+        :type: class
+
         :raise OasisException: If the value is not found and ``required``
             is True
 
         :return: The found value or the default
         """
-        value = getattr(self.args, name, None)
+        # Load order 0:  Get from CLI flag
         source = 'arg'
+        value = getattr(self.args, name, None)
+
+        # Load order 1: ENV override (intended for worker images)
+        if str2bool(os.getenv('OASIS_ENV_OVERRIDE', default=False)) and name != 'config':
```

Comment:
Not sure here, should we check if value is None?
If user specify a value in the MDK, do we want to override it with env variable even if OASIS_ENV_OVERRIDE is True.